src/main/java/br/com/alura/comex/CategoriasProcessadas.java

```diff
@@ -1,6 +1,33 @@
 package br.com.alura.comex;
 
-import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.List;
 
-public class CategoriasProcessadas extends HashSet<String> {
+public class CategoriasProcessadas {
+
+  ArrayList<String> categorias;
+
+  public CategoriasProcessadas() {
+    this.categorias = new ArrayList<>();
+  }
+
+  public List<String> getSortedCategorias() {
+    return this.categorias.stream().sorted((a, b) -> a.compareTo(b)).toList();
```

Comment:
Seria o mesmo que:

```suggestion
    return this.categorias.stream().sorted().toList();
```

O `sorted` usa um lambda parecido com o que vc fez por padrão!